testing/testcafe/tests/dataGrid/editing.ts

```diff
@@ -26,6 +26,8 @@ const getGridConfig = (config): Record<string, unknown> => {
   return config ? { ...defaultConfig, ...config } : defaultConfig;
 };
 
+const encodedIcon = 'data:image/svg+xml,%3Csvg%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20width%3D%2216%22%20height%3D%2216%22%20viewBox%3D%220%200%2016%2016%22%20fill%3D%22none%22%3E%3Cpath%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20d%3D%22M4.913%203.5%209.81%208.397%2012%206.207V12H6.206l2.19-2.19-4.898-4.896L4.913%203.5s0-.028%200%200zM4.914%202c-.384%200-.767.146-1.06.44L2.438%203.852a1.5%201.5%200%200%200%200%202.121L6.275%209.81l-1.982%201.983A1%201%200%200%200%205%2013.5h7.5a.97.97%200%200%200%20.972-.972L13.5%205a1%201%200%200%200-1.707-.707L9.811%206.275%205.975%202.44A1.498%201.498%200%200%200%204.914%202z%22%20fill%3D%22%230067C5%22%3E%3C%2Fpath%3E%3C%2Fsvg%3E';
```

Comment:
Let's make the test icon simpler and more filled
In my opinion, the simple square ideally matches here.

Because it's very intuitive and obvious to see the difference between disabled and enabled square icons:
![65C208D0-6994-4A4F-9BC1-4E86C689D18A_4_5005_c](https://github.com/DevExpress/DevExtreme/assets/48182348/4c617f2f-8268-4789-90ef-e661c81abfe4)

Here is the square base64:
```
data:image/svg+xml;base64,PD94bWwgdmVyc2lvbj0iMS4wIiBlbmNvZGluZz0iaXNvLTg4NTktMSI/Pg0KPCFET0NUWVBFIHN2ZyBQVUJMSUMgIi0vL1czQy8vRFREIFNWRyAxLjEvL0VOIiAiaHR0cDovL3d3dy53My5vcmcvR3JhcGhpY3MvU1ZHLzEuMS9EVEQvc3ZnMTEuZHRkIj4NCjxzdmcgIHdpZHRoPSIyMHB4IiBoZWlnaHQ9IjIwcHgiIHZpZXdCb3g9IjAgMCAyMCAyMCIgZmlsbD0iIzAwMDAwMCIgdmVyc2lvbj0iMS4xIiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciPg0KCTxyZWN0IHdpZHRoPSIxMDAlIiBoZWlnaHQ9IjEwMCUiIC8+DQo8L3N2Zz4NCg==
```